srv/tuto.nit

```diff
@@ -0,0 +1,154 @@
+
+# The base classes with the static knowledge about tutorial and exercises
+module tuto
+
+import markdown
+import ini
+import poset
+
+# The main knowledge about lessons and exercises
+class Tutorial
+	# Dictionary of exercises by the internal name and the title id.
+	var exercise_by_name = new HashMap[String, Exercise]
```

Comment:
* name is the short crappy name used in the config files. It is used to locate file and build dependencies
* title_id is a striped version of the title, it is used in the url and other REST stuff.